Sources/MapboxNavigation/NavigationMapView+CLLocationManagerDelegate.swift

```diff
@@ -1,6 +1,11 @@
 import CoreLocation
 
 extension NavigationMapView: CLLocationManagerDelegate {
+
+    public func locationManager(_ manager: CLLocationManager, didChangeAuthorization status: CLAuthorizationStatus) {
+        if !_locationChangesAllowed { return }
+        authorizationStatus = status
+    }
     
     public func locationManagerDidChangeAuthorization(_ manager: CLLocationManager) {
```

Comment:
Let's mark this method with `@available(iOS 14.0, *)` and leave only:
```swift
authorizationStatus = manager.authorizationStatus
accuracyAuthorization = manager.accuracyAuthorization
```
instead of 
```swift
if #available(iOS 14.0, *) {
    authorizationStatus = manager.authorizationStatus
    accuracyAuthorization = manager.accuracyAuthorization
} else {
    authorizationStatus = CLLocationManager.authorizationStatus()
}
```